tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -322,22 +321,34 @@ def __init__(self, size, stride, padding):
     self.stride = np.asarray(stride)
     self.padding = np.asarray(padding)
 
-  def compute_input_coordinates(self, x, axis=None):
+  def compute_input_center_coordinates(self, x, axis=None):
     """
     Computes the center of the receptive field that generated a feature.
 
     Args:
-      x: Coordinate of the feature.
-      axis: The dimensions for which to compute the input coordinates.
+      x: An array of feature coordinates with shape `(..., d)`, where `d` is the
+        number of dimensions of the coordinates.
+      axis: The dimensions for which to compute the input center coordinates.
+        If `None` (the default), compute the input center coordinates for all
+        dimensions.
 
     Returns:
-      y: Center of the receptive field that generated the feature.
+      y: Center of the receptive field that generated the features.
```

Comment:
What about: "Center of the receptive field that generated the features at the input of the network."